client/OPUNetGameSelectWnd.cpp

```diff
@@ -107,8 +107,10 @@ int OPUNetGameSelectWnd::DlgProc(UINT uMsg, WPARAM wParam, LPARAM lParam)
 		// Get the controlId
 		controlId = wParam;
 		notifyCode = ((NMHDR*)lParam)->code;
-
-		if ((controlId == IDC_GamesList) && (notifyCode == static_cast<UINT>(NM_DBLCLK)))
+#pragma warning( push )
+#pragma warning( disable : 26454 ) // MSVC C26454 produced within expansion of NM_DBLCLK
+		if ((controlId == IDC_GamesList) && (notifyCode == NM_DBLCLK))
+#pragma warning( pop )
```

Comment:
Consider using `suppress` rather than `push`, `disable`, `pop`:
```cpp
#pragma warning(suppress: 26454) // MSVC C26454 produced within expansion of NM_DBLCLK
		if ((controlId == IDC_GamesList) && (notifyCode == NM_DBLCLK))
```

Reference: [warning pragma](https://docs.microsoft.com/en-us/cpp/preprocessor/warning?view=vs-2019)

Reply:
For some reason I was thinking the suppress wouldn't work since the macro was contained in a different header, but with testing locally it is working. Thanks for pointing out. 

I force pushed so the commit message could be updated.